eos-migrate-chrome

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e -x
+
+if ! [[ $(flatpak info -o com.google.Chrome//eos3) == "eos-apps" ]]; then
+  echo "eos-apps Chrome is not installed"
+  exit
+fi
+
+if ! [[ $(flatpak info -o com.google.Chrome//stable) == "flathub" ]]; then
+  echo "Install Flathub Chrome"
+  flatpak install flathub com.google.Chrome -y
+fi
+
+/usr/lib/eos-boot-helper/eos-migrate-chrome-helper --mimeapps
```

Comment:
I only just noticed this. It's not correct to run this from the system service: the mimeapps settings are per-user.

I think you should remove the `--mimeapps` and `--config` flags for `eos-migrate-chrome-helper` and have it always do both.

Reply:
Yes, of course; otherwise you'd change the default browser to a browser that's not installed.

Reply:
ok, goes back to https://github.com/endlessm/eos-boot-helper/pull/422#discussion_r2378604189 and add the symlink as https://github.com/endlessm/eos-boot-helper/pull/422#discussion_r2378749403 simply.

Updated as the new commit.